src/app/containers/EpisodeList/RecentAudioEpisodes/index.jsx

```diff
@@ -113,11 +113,12 @@ const RecentAudioEpisodes = ({ masterBrand, episodes, brandId, pageType }) => {
         ulProps={ulProps}
         liProps={liProps}
       >
-        {episodes.map(episode => (
+        {episodes.map((episode, index) => (
           <EpisodeList.Episode key={episode.id} ref={viewTrackerRef}>
             <EpisodeList.Link
               href={getUrl(episode.id)}
               onClick={clickTrackerHandler}
+              index={`episodeLinkIndex-${index}`}
```

Comment:
Is there a reason why the index is composed as a string here instead of being passed down as an integer/number, and then being composed within the `<EpisodeList.Link>` component? Whenever I see index I always consider it as a number not a string.

Reply:
I'd agree here, I'd do the concatenation in `src/app/containers/EpisodeList/Link.jsx` and pass the index down as an integer. At this level it is just an index, inside the `Link` component it is used as an id so it's clearer why the concatenation is taking place.

Reply:
Yes, thanks for the reviews. I agree with this too and changed accordingly 😄